common/men_at_arms_types/_men_at_arms_types.info

```diff
@@ -36,4 +36,6 @@
 
 	ai_quality = { ... }	# Script value in character scope. Used in addition to a quality calculation (based on TOUGHNESS_SCORE_MULT, etc.) to figure out what the AI should buy/disband
 	fallback_in_hired_troops_if_unlocked = yes # Mercs/holy orders won't have a preference towards this MaA if it is unlocked
+
+	allowed_in_hired_troops = no # Will ban specific MaA from being present in the mercenary pool, the default value is "yes"
```

Comment:
The Scourge still creates bands. What is the difference between using ghouls and archers?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/142989306-f4e7ac35-8bbe-4ecf-86d3-6c5d1fbaa020.png)

</details>

Reply:
I'm not sure if we can outright ban a culture from producing mercs, AFAIK. My goal here was to make sure the special units dont end up in the mercenary band

Reply:
> My goal here was to make sure the special units dont end up in the mercenary band
It's not a problem.